packages/react/src/components/ButtonGroup/README.md

```diff
@@ -22,29 +22,38 @@ import React from 'react';
 import ReactDOM from 'react-dom';
 import { ArrowRight20, ArrowDown20 } from '@carbon/icons-react';
 import { ButtonGroup } from '@carbon/ibmdotcom-react';
+import '@carbon/grid/scss/grid.scss';
 import 'yourapplication.scss';
 import '@carbon/ibmdotcom-styles/scss/components/buttongroup/_buttongroup.scss';
 function App() {
   return;
-  <ButtonGroup
-    buttons={[
-      {
-        href: '',
-        copy: 'Primary action button',
-        renderIcon: ArrowDown20,
-      },
-      {
-        href: '',
-        copy: 'Secondary action button',
-        renderIcon: ArrowRight20,
-      },
-    ]}
-  />;
+  <div className="bx-grid">
+    <div className="row">
+      <div className="bx--col-lg-12 bx--col-md-8 bx--col-sm-16">
+        <ButtonGroup
+          buttons={[
+            {
+              href: '',
+              copy: 'Primary action button',
+              renderIcon: ArrowDown20,
+            },
+            {
+              href: '',
+              copy: 'Secondary action button',
+              renderIcon: ArrowRight20,
+            },
+          ]}
+        />
+        ;
+      </div>
+    </div>
+  </div>;
 }
 ReactDOM.render(<App />, document.querySelector('#app'));
 ```
 
-> 💡 Don't forget to import the button group styles from
+> Utilize the grid system to resize the buttons to fit the desired sizes 💡
+> Don't forget to import the button group styles from
```

Comment:
@andysherman2121 I'm wondering if this needs a better explanation of what's going on here. There's nothing stopping users from using the `ButtonGroup` as-is, the only effect they are getting with the grid wrapper is to let the buttons shrink when the text is smaller. We can have one example without any grid wrapper, and another with it. And in tandem, show the difference between the two (add some text above each example)

Reply:
Sounds good, i'll adjust